udata/core/dataset/api.py

```diff
@@ -615,7 +615,7 @@ def get(self):
                 'title': dataset.title,
                 'acronym': dataset.acronym,
                 'slug': dataset.slug,
-                'image_url': dataset.image_url,
+                'image_url': dataset.organization.logo.filename if dataset.organization else dataset.owner.avatar.filename,
```

Comment:
It should'nt be filename but the full url though.

Reply:
The marshalling will already build the URL it needs only the filename in order to make a valid url.
If passed the full url the result is something like
`https://static.data.gouv.fr/https://static.data.gouv.fr/avatars/8a/4e0caeb30843a08eca1a4a54e025c0-original.png`